software/lobehub.yml

```
@@ -0,0 +1,13 @@
+name: "LobeHub"
+website_url: "https://www.lobehub.com"
+source_code_url: "https://github.com/lobehub/lobe-chat"
+description: "Modern design AI chat framework supporting multiple AI providers, one click install MCP Marketplace and Artifacts / Thinking."
+licenses:
+  - Apache-2.0
```

As discuessed, please update the license to `Proprietary`
```suggestion
  - ⊘ Proprietary
```